pytorch_lightning/core/lightning.py

```diff
@@ -1176,22 +1176,25 @@ def toggle_optimizer(self, optimizer: Optimizer, optimizer_idx: int):
             optimizer: Current optimizer used in training_loop
             optimizer_idx: Current optimizer idx in training_loop
         """
+
```

Comment:
hey can we use the `opt_idx` to get the optimizer down there??... just to unify the arguments with `untoggle_optimizer`

Reply:
n00b q: why do we need it when the current optimizer is already passed in?

Reply:
Nah we don't. either `optimizer` or `optimizer_idx` is enough. `optimizer_idx` seems good for both.